examples/kcp/run-kcp.bash

```
@@ -0,0 +1,45 @@
+    # The APIExportEndpointSlice for the polyglot API and the resource
+    # to broker
+    "-brokerapi=pgs"
+    "-group=example.platform-mesh.io"
+    "-version=v1alpha1"
+    "-kind=PG"
+
+    "-kcp-host-override=127.0.0.1"
+    "-kcp-port-override=8443"
+)
+
+go run ./cmd ${args[@]} "$@"
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)